tests/opkg.py

```diff
@@ -41,3 +41,10 @@ def runTest(self):
         board.expect('All package lists updated')
         board.expect(prompt)
 
+class OpkgInstall(rootfs_boot.RootFSBootTest):
+    '''Opkg is able to install selected packages'''
+    def runTest(self):
+        packages = [ "nano", "vim", "iperf3" ]
```

Comment:
Any reasoning for these packages, perhaps one from each feed would give more coverage?